mlrun/serving/v2_serving.py

```diff
@@ -542,48 +542,67 @@ def _init_endpoint_record(
         function_uri=graph_server.function_uri, versioned_model=versioned_model_name
     ).uid
 
+    model_ep = None
     # If model endpoint object was found in DB, skip the creation process.
     try:
-        mlrun.get_run_db().get_model_endpoint(project=project, endpoint_id=uid)
-
+        model_ep = mlrun.get_run_db().get_model_endpoint(
```

Comment:
seems you are trying get the mep even if tracking is not enabled, why?

Reply:
It is  for the case that the tacking were enabled and now the user is disabling it, so you can see I patch the exist MEP.